notebooks/basics/quantum-circuits.ipynb

```diff
@@ -293,7 +299,7 @@
     "$$\n",
     "</p>\n",
     "\n",
-    "The unitary operation on the qubits $(\\mathsf{X},\\mathsf{Y})$ represented by the entire circuit, to which we will give the name $U$, is therefore obtained by composing the operations:\n",
+    "The unitary operation of the entire circuit, which we'll call '$U$', is the composition of the operations:\n",
```

Comment:
No single-quotes around $U$:

"The unitary operation of the entire circuit, which we'll call $U$, is the composition of the operations:"